build.gradle.kts

```
@@ -1,6 +1,6 @@
 plugins {
-    id("org.openrewrite.build.root") version("latest.release")
-    id("org.openrewrite.build.java-base") version("latest.release")
+    id("org.openrewrite.build.root") version("2.5.6")
+    id("org.openrewrite.build.java-base") version("2.5.6")
```

Once the first build passes we'll want to revert this change, and update the branch to include master, to show that we then get `Object` instead of `TaskContainer`.
```suggestion
    id("org.openrewrite.build.root") version("latest.release")
    id("org.openrewrite.build.java-base") version("latest.release")
```